### ISSUE TITLE
loss and eval

### ISSUE THREAD

**Issue by Friday-HJ** [AndreyFang/ODNet#1]
Hello author, I have utilized your code to train on Megadepth. However, after 20000 iterations, I noticed that my loss did not exhibit a decreasing trend. Could you kindly explain the reason for this?Additionally, I noticed that your evaluation code is not publicly available. Is your evaluation code also based on the CAPS?

![Uploading {42B4D028-5496-4ccc-AE84-B835FE01D05E}.png…]()

**Comment by Friday-HJ** [comment]
![{42B4D028-5496-4ccc-AE84-B835FE01D05E}](https://github.com/AndreyFang/ODNet/assets/41751028/8a18ab81-6871-4daf-b4d0-23e096e34c7e)

**Comment by zhouh02** [comment]
Bro, can you share the pre-training model, training set mentioned in ODNet practice. the link in git is not working. Thanks.
